### PR TITLE
EB release

### PR DESCRIPTION
**Pull request by gabiseabra**
Add release workflow on circleci and document steps to deploy on beanstalk.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,64 +1,156 @@
...
+    environment: *env-production
+
+workflows:
+  commit:
+    jobs:
+      - build
+      - build_release_production:
+          context: boilerplate-production
+          requires:
+            - build
```
Comment (pmargreff) [inline .circleci/config.yml:133]: maybe is a good idea require `test` here as well, so we don't deploy if tests aren't finished...

File: config/releases.exs [hunk 2]
```diff
@@ -12,30 +12,3 @@ config :boilerplate, Boilerplate.Repo,
...
-  level: :info,
-  backends: [:console, LoggerLagerBackend],
-  handle_otp_reports: false,
-  utc_log: true
-
-config :sentry,
-  filter: BoilerplateWeb.SentryEventFilter,
-  dsn: System.fetch_env!("SENTRY_DSN")
-
-config :lager,
```
Comment (pmargreff) [inline config/releases.exs:26]: Does it work? I haven't seen it replaced in any configuration file.

**Review by pmargreff — APPROVED** [review]
LGTM :+1:

File: config/releases.exs (hunk 2 above)
Comment (gabiseabra) [reply config/releases.exs:26]: I've moved these configs to `/config/prod.exs`, should I put it somewhere else?

File: .circleci/config.yml (hunk 1 above)
Comment (gabiseabra) [reply .circleci/config.yml:133]: `test` is required on approval steps which block the ones that actually deploy a release to eb.

File: config/releases.exs (hunk 2 above)
Comment (pmargreff) [reply config/releases.exs:26]: No, I didn't found in the diff, but it looks like it's already there.

**Pull request merged**